frontend/src/app/modules/mips/components/list-mipset-mode/list-mipset-mode.component.scss

```diff
@@ -672,9 +672,9 @@ a {
     border-bottom-color: #748aa1;
   }
 
-  tr:last-child td.mat-cell {
-    border-bottom-color: transparent;
-  }
+  // tr:last-child td.mat-cell {
```

Comment:
Please avoid letting commenting code, if it is required in the future, it is always possible to get the commented code back from Git